### PR TITLE
Add testimonial of 6-way CFD+DEM+FEM simulation at LuXDEM, University of Luxembourg

### PR DESCRIPTION
**Pull request by prasadadhav**
This adds a testimonial for the 6-way CFD+DEM+FEM momentum coupling of the Abrasive Water Jet Cutting Nozzle.
Please let me know if the text is ok, or if there are any suggestions.

### CODE REVIEW

File: collections/_testimonials/24-LuXDEM.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Evaluation of erosion phenomena in a nozzle for Abrasive Water Jet Cutting"
+author: "Prasad ADHAV"
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md]:
```suggestion
author: "Prasad Adhav"
```
Just to be consistent with the other posts.

File: collections/_testimonials/24-LuXDEM.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+author_link: "https://luxdem.uni.lu/team/padhav/"
+organisation: "LuXDEM, University of Luxembourg, LU"
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md]:
```suggestion
organisation: "LuXDEM, University of Luxembourg"
```
Should be clear without this.

File: collections/_testimonials/24-LuXDEM.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+organisation_link: ""
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md]:
```suggestion
organisation_link: "https://luxdem.uni.lu/index.html"
```

File: collections/_testimonials/24-LuXDEM.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+img: testimonial-24-LuXDEM.png
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md:7]: Needs to be the same name as your picture below. Could you please rename the picture?

File: collections/_testimonials/24-LuXDEM.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+---
+In an Abrasive Water Jet Cutting (AWJC) operation a high-speed water jet is used to accelerate abrasive particles forming a turbulent mixture of water, entrained air and abrasive powders traveling at hundreds of meters per second. The focusing tube (Nozzle) represents a key component, whose primary scope is to focus and stabilize the flow forming in the mixing chamber, in order to ensure optimal cutting performances of the device. Nevertheless, this nozzle often happens to be the first target of the erosive action of the flow. This phenomenon significantly shortens the operational life of a nozzle. The numerical approach proposed in this work aims to provide an insight to this very fast and disruptive phenomena that are difficult and expensive to be captured by purely experimental studies.
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md]:
```suggestion
In an Abrasive Water Jet Cutting (AWJC) operation, a high-speed water jet is used to accelerate abrasive particles forming a turbulent mixture of water, entrained air, and abrasive powders traveling at hundreds of meters per second. The focusing tube (Nozzle) represents a key component, whose primary scope is to focus and stabilize the flow forming in the mixing chamber, in order to ensure optimal cutting performances of the device. Nevertheless, this nozzle often happens to be the first target of the erosive action of the flow. This phenomenon significantly shortens the operational life of a nozzle. The numerical approach proposed in this work aims to provide an insight to this very fast and disruptive phenomena that is difficult and expensive to be captured by purely experimental studies.
```

File: collections/_testimonials/24-LuXDEM.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+Such state of the art numerical models have only been able to capture erosion through number of impacts, and do not account for the impact forces. In our prototype, we couple 3 single physics blackbox numerical solvers (software) to create a multiphysics simulation enviornment. Our prototype uses preCICE to couple these 3 numerical solvers: XDEM (for the particle motion), OpenFOAM (for the water jet and entrained air), and CalculiX (for the nozzle deformation). This 6-way coupling between DEM+CFD+FEM brings the simulation of the particle-laden multiphase flow inside the abrasive cutting nozzle close to the real-life conditions. Thus opening up opportunities for further investigation and improvement of the Nozzle design. [Learn more](https://luxdem.uni.lu/research/nozzle_erosion/)
```
Comment (uekerman) [inline collections/_testimonials/24-LuXDEM.md]:
```suggestion
Such state of the art numerical models have only been able to capture erosion through number of impacts, and do not account for the impact forces. In our prototype, we couple three single-physics black-box numerical solvers to create a multiphysics simulation environment. Our prototype uses preCICE to couple the three numerical solvers: XDEM (for the particle motion), OpenFOAM (for the water jet and entrained air), and CalculiX (for the nozzle deformation). This six-way coupling between DEM+CFD+FEM brings the simulation of the particle-laden multiphase flow inside the abrasive cutting nozzle close to real-life conditions -- thus, opening up opportunities for further investigation and improvement of the nozzle design. [Learn more](https://luxdem.uni.lu/research/nozzle_erosion/)
```

**Review by uekerman — CHANGES_REQUESTED** [review]
Thanks a lot :+1:

A few minor tweaks below. Please let me know in case you do not know how to apply changes. We could also do it for you.

File: collections/_testimonials/24-LuXDEM.md (hunk 1 above)
Comment (prasadadhav) [reply collections/_testimonials/24-LuXDEM.md]: Haha, I think the French/Luxembourgish way of writing the names. Now it's muscle memory.

File: collections/_testimonials/24-LuXDEM.md (hunk 4 above)
Comment (prasadadhav) [reply collections/_testimonials/24-LuXDEM.md:7]: Pushed the changed.

File: collections/_testimonials/24-LuXDEM.md (hunk 3 above)
Comment (uekerman) [reply collections/_testimonials/24-LuXDEM.md]: Why not giving the link here?

**Review by uekerman — APPROVED** [review]

**Pull request merged**